recipe/meta.yaml

```diff
@@ -27,9 +27,6 @@ test:
         - humanfriendly
     commands:
         - humanfriendly --help  # [not osx]
-    requires:
-        - capturer >=2.1  # [not win]
-        - coloredlogs >=2.0
```

Comment:
This is only needed to run the test suite of `humanfriendly`. Since we're not doing this here, I decided to remove it.